homeassistant/helpers/update_coordinator.py

```diff
@@ -41,8 +44,15 @@ def __init__(
         self._unsub_refresh: Optional[CALLBACK_TYPE] = None
         self._request_refresh_task: Optional[asyncio.TimerHandle] = None
         self.failed_last_update = False
+        if request_refresh_debouncer is None:
+            request_refresh_debouncer = Debouncer(
```

Comment:
I like this. It makes the coordinator interface a lot easier for the normal case.